src/mpyl/steps/deploy/k8s/chart.py

```diff
@@ -714,7 +716,7 @@ def to_deployment(self) -> V1Deployment:
         liveness_probe, startup_probe = self._construct_probes()
 
         container = V1Container(
-            name=self.release_name,
+            name="service",
```

Comment:
and this?

Reply:
same

Reply:
both of them were like this in MPL, this particular change helps with querying the logs of the service